gradle.properties

```diff
@@ -55,6 +55,7 @@ stdlibMqttVersion=1.1.1
 stdlibOAuth2Version=2.11.0
 stdlibTomlVersion=0.5.1
 stdlibYamlVersion=0.5.3
+stdlibDataCsvVersion=0.1.0
```

Comment:
Shall we use a timestamped version here and check whether the build is passing?

Reply:
Added in https://github.com/ballerina-platform/ballerina-distribution/pull/5528/commits/1546f3612e4d807c6e7ac027bde3e1b180aca879